src/components/CouponDetails/index.jsx

```diff
@@ -479,6 +486,12 @@ class CouponDetails extends React.Component {
 
     this.resetCodeActionStatus();
 
+    if (doesCodeActionHaveErrors) {
+      this.setState({
```

Comment:
It would be great if we can combine this state update with the one below.

Reply:
Done